.circleci/scripts/wait_for_perf_migration_upload_results.py

```diff
@@ -18,11 +16,11 @@ def _wait_for_master() -> None:
     cert = certs.Cert(noverify=True)
     sess = api.UnauthSession("http://127.0.0.1:8080", cert).with_retry(0)
 
-    # 2 hours is the most a migration can take, with this setup.
+    # 15 minutes is the most a migration can take, with this setup.
     # If a migration takes longer than that we have hit an issue a customer will likely hit too.
     time_start = time.time()
     time_last_report = time_start
-    while time_start - time.time() < 2 * 60 * 60:
+    while time_start - time.time() < 15 * 60:
```

Comment:
This line may also be relevant: https://github.com/determined-ai/determined/blob/40a70cf86142d6b35717dfd338c45e6412cd8cd0/.circleci/real_config.yml#L2149